patches/tock/05-mpu-fix.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/kernel/src/sched.rs b/kernel/src/sched.rs
+index ed3ae8260..f109960d3 100644
+--- a/kernel/src/sched.rs
++++ b/kernel/src/sched.rs
+@@ -632,6 +632,7 @@ impl Kernel {
+                     let context_switch_reason = process.switch_to();
+                     scheduler_timer.disarm();
+                     chip.mpu().disable_app_mpu();
++                    chip.mpu().clear_mpu();
```

Comment:
What's the purpose of this patch? Is there a bug tracked upstream?

Reply:
Allistair changed the behavior of the MPU (https://github.com/tock/tock/commit/128782dad5ad410d3e48c16aadba60d9ed6d842b) and this makes our firmware panic when we try to touch the storage.
This one line patch reverts the change made upstream. We need to investigate why we panic and how to properly fix it without diverging from upstream

Reply:
Can you file an upstream bug before we merge the pull request here?

Reply:
I'm not sure it's an upstream bug. Maybe we were relying on the MPU being lenient in the kernel which is not the case anymore. I will open a bug on our side to investigate whether it's a problem upstream or on our side (e.g. maybe we need to add MPU configurations for the storage)

Reply:
Created #155